src/ORM/FieldType/DBForeignKey.php

```diff
@@ -36,9 +34,16 @@ class DBForeignKey extends DBInt
      *
      * @config
      * @var int
+     * @deprecated 5.2.0 Use dropdown_lazyload_threshold instead
      */
     private static $dropdown_field_threshold = 100;
 
+    /**
+     * Number of related objects to show in a dropdown before it switches to using lazyloading
+     * This will also be used as the lazy load limit
+     */
+    private static int $dropdown_lazyload_threshold = 10;
```

Comment:
Since the default threshold will now remain at 100, I don't see any reason to deprecate the old config - let's just keep using it.

Reply:
Not done yet?

Reply:
Updated